js/plugin/popup-dialog.ts

```diff
@@ -526,11 +630,52 @@ export class BrowserModPopup extends LitElement {
 
   static get styles() {
     return css`
-      /* Classes from haStyleDialog */
+      /* Styles adapted from Home Assistant more-info dialog */
```

Comment:
Home Assistant is transitioning to ha-wa-dialog (webawesome) and more-info will be soon. 

If you are willing and game, refactoring your PR to use ha-wa-dialog while including your improvements is teh best way to proceed. Tweaking ha-dialog only to retweak when moving to ha-wa-dialog will be too many changes to support users with. I am not sure if/when ha-dialog will get removed but it will be likely sooner rather than later.